vector/src/main/java/im/vector/app/features/spaces/SpaceListViewModel.kt

```diff
@@ -296,7 +296,7 @@ class SpaceListViewModel @AssistedInject constructor(@Assisted initialState: Spa
             communityGroups
```

Comment:
should rename this to spaceList

Reply:
I agree, but I wanted to limit the number of changes for the hotfix